<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>udg::QListenRisRequestsConfigurationScreenBase</class>
 <widget class="QWidget" name="udg::QListenRisRequestsConfigurationScreenBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>153</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RIS Listener Configuration</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>9</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_textPortListenRisRequests">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="m_checkBoxListenRisRequests">
     <property name="text">
      <string>Listen to RIS requests on port</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="m_RISRequestsPortWarningIcon">
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/images/icons/emblem-warning.svg);</string>
     </property>
     <property name="pixmap">
      <pixmap>:/images/warningIcon.png</pixmap>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="m_RISRequestsPortWarningLabel">
     <property name="text">
      <string>Warning message</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>31</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="m_checkBoxViewAutomaticallyStudies">
       <property name="text">
        <string>Automatically view the studies upon reception</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="4">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_checkBoxListenRisRequests</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_textPortListenRisRequests</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_checkBoxListenRisRequests</sender>
   <signal>toggled(bool)</signal>
   <receiver>m_checkBoxViewAutomaticallyStudies</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
